### DISCUSSION TITLE
视频进度条会莫名其妙跳到最后

### DISCUSSION THREAD

**Discussion by yumuxue** [ywmoyue/biliuwp-lite#867] [bug]
### Bug 描述

打开视频后，播放一段时间的视频，进度条会突然跳转到视频最后，视频本身并未播放完

### 操作步骤

```plain text
1.打开一个时常较长的视频（应该需要超过5min），如BV1LN2YY6En6
2.播放至3min左右
3.在播放过程中视频将直接跳转至最后片尾
```

### 期望结果

正常播放完全

### 截图

https://github.com/user-attachments/assets/a16e146c-54fe-49ff-a110-ea89265de711

### 日志

log
[20241013.log](https://github.com/user-attachments/files/17356103/20241013.log)

设置
[2024-10-13-23_07_58.zip](https://github.com/user-attachments/files/17356098/2024-10-13-23_07_58.zip)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

Windows 10 20H2 (Build 19042)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 备注

这是电脑上的所有程序
![@(YFOW$2{)BVB9BT%P{%MMG](https://github.com/user-attachments/assets/e963e756-7626-465f-8b40-33618714e1ed)
![7 D_L))GOC3NAVF8%D~Z }U](https://github.com/user-attachments/assets/e19120d5-d2ad-412d-b178-b04df8987bb7)
![B94LA4N((1KLZ6F655JA1OF](https://github.com/user-attachments/assets/d6988c41-6f6a-4c02-ba01-d4d90435707f)
![DIH0KM98OM@C0)`C`)Y7M 1](https://github.com/user-attachments/assets/8c6a55b1-cc00-4f50-b0e3-fd60271eaf97)
![M X}0T 1RN{9F }0HSS{_ O](https://github.com/user-attachments/assets/18cb86e9-1e1d-4555-80a0-95dbf9bc4364)

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
@yumuxue 是否已更新最新临时版本https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.4.502

没有的话可以更新试试

**Reply by yumuxue** [reply]
更新临时版复测一个小时左右，暂未发现问题
但是临时版用罗技的logi options+添加快捷键没有反应
![K3YCPE`909UID8$05@KR{NI](https://github.com/user-attachments/assets/0da1ac51-6ac8-414a-bd9f-d85db278095c)
除此之外，应用使用音量合成器调节应用音量，重启应用后系统音量会自动调回默认最大值（4.7.4和最新临时版都存在）
![2024-10-14-11-44-18 - Trim](https://github.com/user-attachments/assets/a440f410-28f7-42ff-bb65-988bce77028e)
